test/end-to-end/offer.js

```diff
@@ -633,41 +632,139 @@ describe("Offer endpoint tests", () => {
             const test_offer_1 = {
                 title: "Test Offer 1",
                 publishDate: "2021-01-14T00:00:00.000Z",
-                publishEndDate: "20-02-28T00:00:00.000Z",
+                publishEndDate: "2021-02-28T00:00:00.000Z",
                 description: "For Testing Purposes",
                 contacts: ["geral@niaefeup.pt", "229417766"],
                 jobType: "SUMMER INTERNSHIP",
                 fields: ["DEVOPS", "MACHINE LEARNING", "OTHER"],
                 technologies: ["React", "CSS"],
+                isHidden: false,
                 // owner: Will be set in beforeAll,
                 location: "Testing Street, Test City, 123",
             };
             const test_offer_2 = {
                 title: "Test Offer",
-                publishDate: "2019-11-22T00:00:00.000Z",
-                publishEndDate: "2019-11-28T00:00:00.000Z",
+                publishDate: "2021-01-14T00:00:00.000Z",
+                publishEndDate: "2021-02-28T00:00:00.000Z",
                 description: "For Testing Purposes",
                 contacts: ["geral@niaefeup.pt", "229417766"],
                 jobType: "SUMMER INTERNSHIP",
                 fields: ["FRONT_END", "BACK_END"],
                 technologies: ["Flutter", "Node.js"],
+                isHidden: false,
                 // owner: Will be set in beforeAll,
                 location: "Testing Avenue, Test State, 321",
             };
```

Comment:
These date changes will "expire" after some time, you need a test_offer generation function that uses a valid date every time. We can sync in PM about this